src/main/java/com/openshift/internal/restclient/URLBuilder.java

```diff
@@ -105,7 +104,7 @@ public URLBuilder resource(IResource resource) {
 	}
 
 	public URLBuilder addParmeter(String key, String value) {
-		params.put(key, value);
+		params.add(new AbstractMap.SimpleEntry<>( key, value ));
```

Comment:
wfm as long as its backwards compatible. I am unable to think of any place this might be an issue...but..just in case